### PR TITLE
feat(); Baseswap Base; Add Baseswap Base Subgraph

### PR DESCRIPTION
**Pull request by steegecs**
**Context:**
This PR integrates BaseSwap Base subgraph into the Uniswap Forks folder. This is a simple integration and works just like Uniswap V2. 

**Note:**
The protocol collects all of the fees (0.25% of swap). The LPs are reimbursed through staking in the MasterChef contract. This PR is not yet going to account for rewards earned through the MasterChef contract. Just the core functionality of the DEX.

### CODE REVIEW

File: deployment/deployment.json [hunk 1]
```diff
@@ -5151,6 +5151,36 @@
       }
     }
   },
+  "baseswap": {
+    "schema": "dex-amm",
+    "base": "uniswap-forks",
+    "protocol": "baseswap",
+    "project": "ubeswap",
```
Comment (dhruv-chauhan) [inline deployment/deployment.json:5158]: `"project": "baseswap",`

File: deployment/deployment.json [hunk 2]
```diff
@@ -5151,6 +5151,36 @@
...
+      "baseswap-base": {
+        "network": "base",
+        "status": "prod",
+        "versions": {
+          "schema": "1.3.2",
+          "subgraph": "1.0.0",
+          "methodology": "1.0.0"
+        },
+        "files": {
+          "template": "uniswap.v2.template.yaml"
```
Comment (dhruv-chauhan) [inline deployment/deployment.json:5169]: change template file name to `baseswap.template.yaml`

File: subgraphs/uniswap-forks/src/versions.ts
Comment (dhruv-chauhan) [inline subgraphs/uniswap-forks/src/versions.ts:1]: does not need to be committed

File: subgraphs/uniswap-forks/protocols/baseswap/config/deployments/baseswap-base/configurations.ts [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  }
+  getFactoryAddress(): string {
+    return "0xfda619b6d20975be80a10332cd39b9a4b0faa8bb";
+  }
+  getFactoryContract(): Factory {
+    return Factory.bind(
+      Address.fromString("0xfda619b6d20975be80a10332cd39b9a4b0faa8bb")
+    );
+  }
+  getTradeFee(blockNumber: BigInt): BigDecimal {
```
Comment (dhruv-chauhan) [inline subgraphs/uniswap-forks/protocols/baseswap/config/deployments/baseswap-base/configurations.ts:47]: disable lint,
`// eslint-disable-next-line @typescript-eslint/no-unused-vars`

File: deployment/deployment.json (hunk 1 above)
Comment (steegecs) [reply deployment/deployment.json]: Fixed!

File: deployment/deployment.json (hunk 2 above)
Comment (steegecs) [reply deployment/deployment.json]: changed

File: subgraphs/uniswap-forks/src/versions.ts
Comment (steegecs) [reply subgraphs/uniswap-forks/src/versions.ts:1]: removed

**Review by dhruv-chauhan — APPROVED** [review]
🚀

**Comment by dmelotik** [comment]
Thanks for reviewing @dhruv-chauhan

**Pull request merged**